### DISCUSSION TITLE
Meeting to Jeffrey Lim (Pulse)

### DISCUSSION THREAD

**Discussion by BKVad1m** [Vad1mCSO2/blackknighttracker#3] [General]
I meet to @jthlim . I want to anniversary to Impulse Tracker developer who scripted as full assembly. I tested the Impulse Tracker on My modern actual PC, Setups: SBEMU with EMS, ITSB16.MMX Driver for MMX CPU support, USBDDOS for lag-less input under FreeDOS. Current Jeffrey Lim developing steno keyboard systems. Also X [In Russia Prohibited {Makes impossible meeting}] followed this.
There's video attachments how IT still works
https://drive.google.com/file/d/1VRps9YCwugQLO6qHlGcgpRtv4RzVgR0j/view?usp=sharing
https://drive.google.com/file/d/10Ln_Tn7kFU1vnc0l-oSNydfLBbAxct86/view?usp=drive_link

**Comment by morbing** [comment]
what the fuck

**Comment by BKVad1m** [comment]
https://github.com/jthlim/impulse-tracker
